### PR TITLE
Add queue reference documentation.

### PR DESCRIPTION
**Pull request by tjholm**

### CODE REVIEW

File: docs/reference/project/queues.mdx [hunk 1]
```diff
@@ -1 +1,16 @@
-TODO...
+A Nitric queue is described by its definition in a `nitric.yaml` file. A queue represents a category or tasks that are aysnchronously processed by a service.
```
Comment (raksiv) [inline docs/reference/project/queues.mdx:1]: minor typo - adjust to 'asynchronously'

File: docs/reference/project/queues.mdx [hunk 2]
```diff
@@ -1 +1,16 @@
...
+
+## Properties
+
+> Queues do not have properties at present and are represented only by their name at this time. This is subject to change as requirements around queue configuration emerge
```
Comment (raksiv) [inline docs/reference/project/queues.mdx:5]: Add full stop at end of sentence for consistency with other config defs.

**Review by davemooreuws — APPROVED** [review]
Approved for when minor typo is fixed

File: docs/reference/project/queues.mdx (hunk 1 above)
Comment (tjholm) [reply docs/reference/project/queues.mdx]: fixed

File: docs/reference/project/queues.mdx (hunk 2 above)
Comment (tjholm) [reply docs/reference/project/queues.mdx]: fixed.

**Pull request merged**